### DISCUSSION TITLE
Duda entrega kata.

### DISCUSSION THREAD

**Discussion by LuisAcevedo9** [LaunchX-InnovaccionVirtual/CursoIntroPython#192] [Q&A] [answered]
Disculpen compañeros :c alguien sabe si van a subir la sesión de ayer? o como se realizaría la entrega de las katas :c es que se me fue la luz en mi casa y no pude asistir :c

**Comment by JossPerezT** [comment, marked as answer]
Solo debes subir tus katas a tu repositorio de GitHub. 
Recuerda subirlas en público. 

Cuando Fer nos suba en forms, ahí lo llenas con tu nombre y el link de tu repo. Luego te llegará un correo y te indicara el link para la siguiente misión. 


Y hoy a las 12pm suben las demás katas.

**Reply by OmarAlex24** [reply]
Hola una pregunta, es que tampoco estuve en la sesion de ayer, no sabes si a fuerza tenemos que crear una carpeta que digas mis katas y ahi meter todas, o simplemente podemos clonar el repositorio, modificar el archivo .ipynb y ya?, gracias

**Reply by HectordAponte** [reply]
Hola, compañeros:
Soy de inscripción tardía por lo que me estoy poniendo al día con las grabaciones y las katas. Lo que no me queda claro es cuando es la fecha límite para enviar las katas a fin de calificar a la siguiente misión. En una plática, Fernanda menciona que mañana domingo 13-feb. Pero en otra ocasión, dice que el formulario va a estar activo una semana más, pero entiendo que ya estaría en curso la misión de Front End. ¿Alguien me pudiera aclarar esto, por favor? Muchas gracias.

**Reply by aguilarlima** [reply]
El enlace estará disponible desde el lunes 14 hasta el próximo lunes 21, solo que entre más tardes ese mismo tiempo te atrasarás con la siguiente misión.

**Reply by JorgeRodriguez1998** [reply]
Hola Héctor, buen día. El formulario estará activo a partir del día lunes 14 y ese mismo día será la primer sesión de Front-End a las 8:00 p.m. Puedes enviar las katas el día lunes 14 para entrar a partir de ese día, sin embargo, tienes hasta el siguiente lunes para entregarlas, lo único que pasaría es que vas a comenzar el curso de Front-End con una semana de retraso

Saludos

**Reply by HectordAponte** [reply]
Es decir, no hay problema en que use la siguiente semana para enviar mis katas, pero no me soltarán el enlace a la siguiente misión hasta que las envíe , ¿es así? Lo que me preocupaba es que me dieran de baja si no enviaba algo mañana. Gracias.

**Reply by HectordAponte** [reply]
Gracias, @aguilarlima y @JorgeRodriguez1998. Entiendo entonces que si no termino mañana, no tendré acceso a la primera sesión de Front End, pero siempre podré verla grabada y ponerme al corriente, ¿verdad? Saludos.

**Reply by JorgeRodriguez1998** [reply]
Efectivamente Héctor, las sesiones quedan grabadas para que te pongas al corriente una vez tengas el acceso

¡Mucho éxito!

**Reply by HectordAponte** [reply]
Gracias, Jorge, por tu apoyo. ¡Qué tengas buen día!

**Comment by ArielCanedo** [comment]
Buen día Luis, en team dale click al link de la sesión y se abrirá la
grabación, ahí podrás verla de nuevo

**Comment by LuisAcevedo9** [comment]
Vale vale :) muchas gracias a ambos :)

**Comment by DanielOe21** [comment]
@LuisAcevedo9 Todas las sesiones quedan grabadas, para verlas sólo entra al mismo link de la sesión y dale en reproducir.

Aún no dan el link del formulario para cargar el link de las Katas, por ahora puedes juntarlas en una carpeta en tu computadora y después subirlas a un repositorio de GitHub.

El entregable será el **link de TU repositorio** con las Katas, el formato del link debe ser 
```
github.com/USUARIO/REPOSITORIO
```

Por favor no olvides marcar tu pregunta como CONTESTADA si así lo consideras.

### Follow me!
<a href="https://github.com/DanielOe-21">
<img alt="GitHub followers" src="https://img.shields.io/github/followers/DanielOe-21?style=social">
</a>

**Comment by Nocturnal-OFC** [comment]
¿Entonces no hay link?

**Reply by DanielOe21** [reply]
De la grabación, SI
De entregar las Katas, AUN NO

**Reply by Nocturnal-OFC** [reply]
De acuerdo, muchas gracias

**Reply by lopezArmando** [reply]
Si, hay, pero no esta aun publicado como stream del Equipo Launch X (tal como las anteriores secciones)
Puedes accesar desde la ultima publicación de Fer donde dice:  **SOY EL ENLACE A LA SESIÓN - ENTREGABLES**

https://teams.microsoft.com/l/meetup-join/19%3ameeting_Yjg2N2NlOWYtNTgyNi00Y2JlLWJmOWQtYjAyZDYwYjViODli%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%227b10324a-57d4-43ce-95fe-68f52b498664%22%2c%22IsBroadcastMeeting%22%3atrue%7d

**Comment by aguilarlima** [comment]
Hola, aquí está el vinculo de la sesión de ayer
[SESIÓN ENTREGABLES](https://teams.microsoft.com/l/meetup-join/19%3ameeting_Yjg2N2NlOWYtNTgyNi00Y2JlLWJmOWQtYjAyZDYwYjViODli%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%227b10324a-57d4-43ce-95fe-68f52b498664%22%2c%22IsBroadcastMeeting%22%3atrue%7d)

**Comment by aguilarlima** [comment]
De las dos formas está bien, puedes crear un repositorio en tu perfil de Github, ahí subir tus katas y ese será el enlace, o también puedes duplicar las katas y resolverlas desde ahí, en lo particular me gusta subir las propias.

**Comment by Nocturnal-OFC** [comment]
Entonces no subieron el link para entregar las katas?

**Reply by DanielOe21** [reply]
Compañero @Nocturnal-OFC  si tu pregunta es enserio te sugiero que busques y hagas algunos ejercicios de comprensión lectora.

Ya se ha respondido arriba que el link va a estar disponible a partir del LUNES y hoy es SÁBADO 😁

**Reply by Nocturnal-OFC** [reply]
Gracias por la sugerencia, sin embargo solo era una duda, tal vez no me fijé muy bien lo que dijeron arriba porque estaba un poco ocupado; igualmente gracias

**Reply by DanielOe21** [reply]
Es normal estar ocupado, pero otra buena habilidad a practicar es poder diferenciar lo importante y lo urgente, si estabas ocupado podría ser buena idea esperar a un momento de disponibilidad para revisar la información a detalle.

Igual deduzco que no viste el Live del viernes ni la grabación porque justo de eso trató
